modules/ROOT/pages/authentication-authorization/dbms-administration.adoc

```diff
@@ -68,9 +68,9 @@ All DBMS privileges are relevant system-wide.
 Like user management, they do not belong to one specific database or graph.
 For more details on the differences between graphs, databases, and the DBMS, refer to link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/introduction/cypher_neo4j/[Cypher Manual -> Cypher and Neo4j].
 
-image::privileges_grant_and_deny_syntax_dbms_privileges.svg[width="800", title="Syntax of GRANT and DENY DBMS Privileges"]
+image:privileges_grant_and_deny_syntax_dbms_privileges.svg[title="Syntax of GRANT and DENY DBMS Privileges"]
 
-image::privileges_hierarchy_dbms.svg[width="800", title="DBMS privileges hierarchy"]
+image:privileges_hierarchy_dbms.svg[title="DBMS privileges hierarchy"]
```

Comment:
```suggestion
image::privileges_hierarchy_dbms.svg[title="DBMS privileges hierarchy"]
```

Reply:
Doesn't work with double columns.

Reply:
Also, they weren't clickable before as well.

Reply:
Interesting, why it doesn't work with double colons.
What if to use the following syntax:
````
.Syntax of GRANT and DENY DBMS Privileges.
image::privileges_grant_and_deny_syntax_dbms_privileges.svg[]
````

Reply:
Have you tried it? I can build it locally to see if this will render.

Reply:
no, I haven’t tried it here. I used this syntax in the GSG. It works there

Reply:
Yes, the syntax works for all other images but these, hence the PR and the request.

Reply:
I couldn't figure out why these images were not rendering while all other images in the Operations Manual were. That's what the card is about. When I removed the second column, they showed up.

Reply:
What you are suggesting is what it was before.

Reply:
I thought the problem was in the attr `width=800`

Reply:
try the following: 
```
.Syntax of GRANT and DENY DBMS Privileges
image:privileges_grant_and_deny_syntax_dbms_privileges.svg[role="popup-link"]


.DBMS privileges hierarchy
image:privileges_hierarchy_dbms.svg[role="popup-link"]
```

Reply:
images will be rendered and they'll become clickable. But their captions will be turned into titles

Reply:
I found this Asciidoctor docs: 
`An inline image is displayed in the flow of another element, such as a paragraph or sidebar block. The inline image macro is almost identical to the block image macro, except its macro name is followed by a single colon (:).`
https://docs.asciidoctor.org/asciidoc/latest/macros/images/#inline-image-macro

Reply:
> images will be rendered and they'll become clickable. But their captions will be turned into titles

These are not captions but hover-overs, so we can still keep them.

Reply:
I tried with `::` again. It renders the captions properly (Figure 1. Syntax of GRANT and DENY DBMS privileges) but no image.
I am trying to understand why it assumes that these are inline images.

Reply:
> > images will be rendered and they'll become clickable. But their captions will be turned into titles
> 
> These are not captions but hover-overs, so we can still keep them.

Actually, if they had worked correctly, we wouldn't have needed the titles. The `title=".."` turns into a caption.